src/code-editor/setup-editor.ts

```diff
@@ -102,6 +66,53 @@ export function setupEditor(
     }
   });
 
+  // HACK: Annotations aren't cleared when editor is empty.
```

Comment:
Why aren't the cleared? Are there any references which helps future readers understand why the annotations aren't cleared when editor is empty?

Reply:
I don't know actually, the code was copied as is